### ISSUE TITLE
Can't reset char in dm's, plus some other things

### ISSUE THREAD

**Issue by JcynR** [drizzle-mizzle/CharacterAI-Discord-Bot#35]
1. It's currently impossible to run the ``reset`` cmd and others through dm:
```
⚠ Failed to execute command: Object reference not set to an instance of an object. (Exception)
```

2. Bot should not reply to user/last message in private rooms or dm's:
![image](https://user-images.githubusercontent.com/22550885/221714894-54e4dfd4-5473-4482-b63f-c1bbe8e45973.png)

3. Remove ``amode`` cmd and enable it by default. The bot tends to call the user by the name in the c.ai account profile instead of the Discord name with amode off.

ver 1.82

**Comment by drizzle-mizzle** [comment]
> It's currently impossible to run the `reset` cmd and others through dm

Most of the commands are just not designed to be used out of the server context. I don't intend to change it, sorry. Gotta add some "stubs" for it, so it won't confuse users with these scary errors.
But the `reset` command will be fixed, that's for sure a bug.
> Bot should not reply to user/last message in private rooms or dm's:
It's designed to be like this. Because you can, accidentally or intentionally, send more than one message to a character, and **it will reply on both in parallel**.
I don't know how it does actually works, and how AI processes this kind of mixed context, but it's not restricted to answer on multiple messages at once. The only restrictions are the javascipt animations and text field locks on the main website, which can't stop you in Discord lol. Let it be like this.
> Remove amode cmd and enable it by default. The bot tends to call the user by the name in the c.ai account profile instead of the Discord name with amode off.

No. It doesn't always work flawlessly. Sometimes AI gets very confused because of it. You can enable needed audience mode setting to be used by default in Config file.

**Comment by drizzle-mizzle** [comment]
- Fixed in [1.83](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/releases/tag/1.83)
